src/UI/StructureInspector.cpp

```diff
@@ -124,34 +124,40 @@ void StructureInspector::update()
 	if (!visible()) { return; }
 	Window::update();
 
-	Renderer& r = Utility<Renderer>::get();
+	Renderer& renderer = Utility<Renderer>::get();
+
+	const auto drawTitleText = [&renderer](NAS2D::Point<int> position, std::string title, std::string text) {
```

Comment:
Consider changing the `std::string` parameters to `const std::string&` or `std::string_view` because the function doesn't change them.